.github/workflows/action.yml

```diff
@@ -0,0 +1,20 @@
+name: Github Action
+on: [issues, pull_request]
+jobs:
+  github-actions-automate-projects:
+    runs-on: ubuntu-latest
+    steps:
+    - name: add-new-issues-to-repository-based-project-column
+      uses: docker://takanabe/github-actions-automate-projects:v0.0.1
+      if: github.event_name == 'issues' && github.event.action == 'opened'
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        GITHUB_PROJECT_URL: https://github.com/orgs/creativecommons/projects/10
+        GITHUB_PROJECT_COLUMN_NAME: Pending Review
```

Comment:
Do you need a column name here? If not, just skip this; the project will determine the column name.

Reply:
@kgodey Yea actually, because it specifies to which column, the PR's are need to be placed!